bigquery/tables.rb

```diff
@@ -18,10 +18,9 @@ def create_table project_id:, dataset_id:, table_id:
   # dataset_id = "ID of the dataset to create table in"
   # table_id   = "ID of the table to create"
 
-  require "google/cloud"
+  require "google/cloud/bigquery"
 
-  gcloud   = Google::Cloud.new project_id
-  bigquery = gcloud.bigquery
+  bigquery = Google::Cloud::Bigquery.new
```

Comment:
project: project_id is necessary because this is inside a function which has a project_id parameter.